packages/components/src/DataTable/DataTable.mdx

```diff
@@ -0,0 +1,550 @@
+---
+name: DataTable
+menu: Patterns
```

Comment:
realized this is in the wrong category, you'll want this in the components section :) 
<img width="248" alt="image" src="https://user-images.githubusercontent.com/62039455/198377393-2cff6097-93e3-42c8-8610-3a0523c07453.png">

Reply:
✅